react/Autosuggest/Autosuggest.less

```diff
@@ -49,7 +49,7 @@
   line-height: @sectionTitleHeight;
   height: @sectionTitleHeight;
   margin: @row-height (@gutter-width * 0.75) 0;
-  color: darken(@sk-highlight, 10%);
+  color: @sk-blue-lighter;
```

Comment:
Thoughts on making this `@sk-form-accent`?

Reply:
Yeah, that's a good idea.

Reply:
Done.